app/src/test/java/com/volchok/rocketapp/app/domain/ObserveNavigationEventsUseCaseTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.volchok.rocketapp.app.domain
+
+import com.volchok.rocketapp.app.model.NavigationEvent
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+
+internal class ObserveNavigationEventsUseCaseTest {
+
+    private val mainNavigationController = mockk<MainNavigationController>()
+
+    @Test
+    fun `should navigate to the correct screen`() = runTest {
+        val navigationEvent = mockk<NavigationEvent>()
+
+        every { mainNavigationController.navigationEvent } returns flowOf(navigationEvent)
+
+        val observeNavigationEventsUseCase =
+            ObserveNavigationEventsUseCase(mainNavigationController)
+
+        val result = observeNavigationEventsUseCase.invoke(Unit)
+
+        result.first() shouldBe navigationEvent
```

Comment:
pro vsechny testy plati, ze neni potreba mezi kazdym radkem nechavat newline

delame to tak ze newline davame jen mezi logicke casti testu. tzn za nastaveni mocku a za provolani testovane metody

v tomto pripade by to bylo takto:
```suggestion
    fun `should navigate to the correct screen`() = runTest {
        val navigationEvent = mockk<NavigationEvent>()
        every { mainNavigationController.navigationEvent } returns flowOf(navigationEvent)

        val observeNavigationEventsUseCase =
            ObserveNavigationEventsUseCase(mainNavigationController)
        val result = observeNavigationEventsUseCase.invoke(Unit)

        result.first() shouldBe navigationEvent
```

Reply:
Opraveno